config/config_handler.go

```diff
@@ -171,6 +187,11 @@ func (ch *pdConfHandler) run() {
 	for {
 		select {
 		case <-time.After(ch.interval):
+			if !ch.registered {
+				ch.register()
+				continue
+			}
+
```

Comment:
`pdConfHandler` will call run() when it is not registered?

Reply:
We don't know since it is determined how `PD` implement interfaces `register()` and `get()`.
According to the protocol designed in our proposal, it's better to not allow TiDB call `get()` before `register()`, so I add this logic for safety.